aphrodite/engine/args_tools.py

```diff
@@ -24,96 +24,102 @@
-    load_in_smooth: bool = False
-    deepspeed_fp_bits: Optional[int] = None
-    enforce_eager: Optional[bool] = True
-    max_context_len_to_capture: Optional[int] = None
-    max_seq_len_to_capture: int = 8192
-    disable_custom_all_reduce: bool = False
+    # Tokenizer Options
+    skip_tokenizer_init: bool = False
     tokenizer_pool_size: int = 0
     # Note: Specifying a tokenizer pool by passing a class
     # is intended for expert use only. The API may change without
     # notice.
     tokenizer_pool_type: Union[str, Type["BaseTokenizerGroup"]] = "ray"
     tokenizer_pool_extra_config: Optional[dict] = None
-    enable_lora: bool = False
-    max_loras: int = 1
-    max_lora_rank: int = 16
-    enable_prompt_adapter: bool = False
-    max_prompt_adapters: int = 1
-    max_prompt_adapter_token: int = 0
-    fully_sharded_loras: bool = False
-    lora_extra_vocab_size: int = 256
-    long_lora_scaling_factors: Optional[Tuple[float]] = None
-    lora_dtype: str = "auto"
-    max_cpu_loras: Optional[int] = None
-    device: str = "auto"
-    ray_workers_use_nsight: bool = False
-    num_gpu_blocks_override: Optional[int] = None
-    num_lookahead_slots: int = 0
-    model_loader_extra_config: Optional[dict] = None
-    ignore_patterns: Optional[Union[str, List[str]]] = None
-    preemption_mode: Optional[str] = None
-    # Scheduler config
+    # Scheduler Options
+    use_v2_block_manager: bool = False
     scheduler_delay_factor: float = 0.0
     enable_chunked_prefill: bool = False
     guided_decoding_backend: str = 'outlines'
-    # Speculative decoding config
+    # Speculative Decoding Options
+    num_lookahead_slots: int = 0
     speculative_model: Optional[str] = None
-    speculative_draft_tensor_parallel_size: Optional[int] = None
     num_speculative_tokens: Optional[int] = None
     speculative_max_model_len: Optional[int] = None
-    speculative_disable_by_batch_size: Optional[int] = None
     ngram_prompt_lookup_max: Optional[int] = None
     ngram_prompt_lookup_min: Optional[int] = None
+    speculative_draft_tensor_parallel_size: Optional[int] = None
+    speculative_disable_by_batch_size: Optional[int] = None
     spec_decoding_acceptance_method: str = 'rejection_sampler'
     typical_acceptance_sampler_posterior_threshold: Optional[float] = None
     typical_acceptance_sampler_posterior_alpha: Optional[float] = None
     disable_logprobs_during_spec_decoding: Optional[bool] = None
-
+    # LoRA Options
+    enable_lora: bool = False
+    max_loras: int = 1
+    max_lora_rank: int = 16
+    lora_extra_vocab_size: int = 256
+    lora_dtype: str = "auto"
+    max_cpu_loras: Optional[int] = None
+    long_lora_scaling_factors: Optional[Tuple[float]] = None
+    fully_sharded_loras: bool = False
     qlora_adapter_name_or_path: Optional[str] = None
+    # Soft Prompt Options
+    enable_prompt_adapter: bool = False
+    max_prompt_adapters: int = 1
+    max_prompt_adapter_token: int = 0
+    # API Options
```

Comment:
API options are defined in aphrodite/endpoints/openai/args.py. These are still engine-related args.